src/mesh/Mesh.cpp

```diff
@@ -213,7 +213,9 @@ void Mesh::allocateDataValues()
 void Mesh::computeBoundingBox()
 {
   PRECICE_TRACE(_name);
-  BoundingBox bb(_dimensions);
+
+  BoundingBox bb = _boundingBox.empty() ? BoundingBox(_dimensions) : BoundingBox(_boundingBox);
```

Comment:
Better would probably be to not call "computeBB" is this case.

Reply:
_hm_ should be kept for compatibility reasons. See 

https://github.com/DavidSCN/precice/blob/b9e611c50605783a41a878781229bda624209ab8/src/precice/impl/SolverInterfaceImpl.cpp#L1491-L1492